templates/bot/ts/notification/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "echobot",
```

Comment:
Nit: maybe rename the name field.

Reply:
Can we refine the scaffold code at last? The template is still a draft, not finalized yet.